Makefile.toml

```
@@ -184,6 +184,23 @@ BOOT_CONFIG_INPUT = "${BUILDSYS_ROOT_DIR}/bootconfig-input"
 # Boot Configuration initrd
 BOOT_CONFIG = "${BUILDSYS_ROOT_DIR}/bootconfig.data"
 
+# Determines the kubeconfig that should be used by testsys. If no kubeconfig was provided and the
+# default kubeconfig location does not exist, use the users default kubeconfig.
+CARGO_MAKE_TESTSYS_KUBECONFIG_ARG = {script = [
+   '''
+      if [[ -n ${TESTSYS_KUBECONFIG_PATH} ]]; then
```

style nits: quote all shell variables and avoid `[[` if `[` will do:
```suggestion
      if [ -n "${TESTSYS_KUBECONFIG_PATH}" ]; then
```